power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/y_bus.hpp

```diff
@@ -427,6 +511,17 @@ template <bool sym> class YBus {
 
     // cache the math parameters
     std::shared_ptr<MathModelParam<sym> const> math_model_param_;
+
+    // cache the branch and shunt parameters in sequence_idx_map
+    IdxVector branch_param_idx_{};
+    IdxVector shunt_param_idx_{};
+
+    // map index between admittance entries and parameter entries
+    std::vector<IdxVector> map_admittance_param_branch{};
+    std::vector<IdxVector> map_admittance_param_shunt{};
```

Comment:
we have the convention to always suffix member variables with `_` to clearly make the distinction between persistent and non-persistent data

```suggestion
    // map index between admittance entries and parameter entries
    std::vector<IdxVector> map_admittance_param_branch_{};
    std::vector<IdxVector> map_admittance_param_shunt_{};
```

Reply:
Resolved in latest commit.